docs/source/reporting.rst

```diff
@@ -0,0 +1,68 @@
+.. -*- mode: rst -*-
+.. vi: set ft=rst sts=4 ts=4 sw=4 et tw=79:
+
+.. _chap_reporting:
+
+*********
+Reporting
+*********
+
+When you trigger an error, we want to make it easy to ask for help. By adding
+an integration, helpme_, DataLad can automatically send a traceback of your error
+to datalad_helpme_ where the development team can help you out.
+
+Setup
+=====
+
+To use helpme, you should have the full install of Datalad, or install helpme
+separately:
+
+.. code-block:: console
+
+    $ pip install helpme[github]
+
+
+To submit to GitHub, HelpMe only requires requests, which is already required by
+Datalad, so the install above will not add any additional dependencies other
+than the helpme library itself. If you don't choose to
```

Comment:
If there is no additional dependencies if helpme, I would be ok to make it party of the default installation. Let's see what others say though